### PR TITLE
Handle display inputs

### PR DESCRIPTION
**Pull request by DuFF14**
Utilizes new DisplayConfig API calls to handle multiple video inputs.

### CODE REVIEW

File: OSVR-Unity/Assets/OSVRUnity/src/Math.cs [hunk 1]
```diff
@@ -55,10 +55,28 @@ public static Matrix4x4 ConvertPose(OSVR.ClientKit.Pose3 pose)
...
+                }
+                else
+                {
+                    if(eyeIndex == 0)
+                    {
+                        return new Rect(0f, 0f, 1f, 1f);
+                    }
+                    else
+                    {
+                        return new Rect(0.5f, 0f, 1f, 1f);
```
Comment (DuFF14) [inline OSVR-Unity/Assets/OSVRUnity/src/Math.cs]: Concerned about the hard-coded values here. Is there a better way to handle this?

Comment (JeroMiya) [inline OSVR-Unity/Assets/OSVRUnity/src/Math.cs]: Could you get the display input dimensions for each viewer->eye->surface and transform its viewport from pixel space into 0.0f->1.0f space? There should be a method to get the display input index of a given viewer->eye->surface, which you can then use to get the display input size.

**Comment by DuFF14** [comment]
Thanks for the comment. I've been working on simplifying this and testing the various dSight configurations (landscape 1 or 2 inputs, portrait 1 or 2 inputs). Was seeing some confusing results that I attempted to correct in the code above by checking the numDisplayInputs and eyeIndex when setting viewports. Discovered some issues with the display descriptor that explain some of the weird results I was seeing, so I'll correct those and then come back with some code that works here.
https://github.com/OSVR/OSVR-Core/issues/290
https://github.com/OSVR/OSVR-Core/issues/291

**Comment by DuFF14** [comment]
I think these changes are closer. dSight landscape 1 and 2 inputs look ok. Portrait modes will require display rotation, but I was able to get 2 input portrait mode working by rotating the display in windows and changing the config resolution. Having some issues with 1 input portrait, will revisit tomorrow.

**Comment by DuFF14** [comment]
Added a line to set fullscreen resolution only if we have one video input. If we have multiple inputs and need a double-wide (3840x1080) Unity window, fullscreen resolution gets turned off or else the Unity window confines itself to 1920x1080.

I've tested this in all dSight configurations (and HDK). Until display rotation is part of the display descriptor schema, portrait modes (when the desktop is sideways on the HMD display) will require manually rotating the display in Windows. Also note that if you want Unity to open a 1080x1920 resolution, you need to hide the resolution dialog, or else you'll be forced into a resolution that Unity prefers. More discussion here: http://answers.unity3d.com/questions/309819/getting-unity-to-render-across-multiple-monitors.html
Merging.

**Pull request merged**